boa/environment.py

```diff
@@ -403,6 +404,7 @@ class Env:
     _fast_mode_enabled = False
     _fork_mode = False
     _fork_try_prefetch_state = False
+    _predefined_blocks = {"safe", "latest", "finalized", "pending", "earliest"}
```

Comment:
this could be a module-level constant

Reply:
Done and moved constant and usage to fork.py